File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/RURSellComparator.java
package fobo66.exchangecourcesbelarus.util.comparators;

import fobo66.exchangecourcesbelarus.models.Currency;

/**
 * Created by fobo66 on 31.03.2016.
 */
public class RURSellComparator implements CurrencyComparator {
    @Override
    public int compare(Currency lhs, Currency rhs) {
        return rhs.rurSell.compareToIgnoreCase(lhs.rurSell);
    }
}

File: app/src/main/java/fobo66/exchangecourcesbelarus/util/comparators/RurSellComparator.kt
package fobo66.exchangecourcesbelarus.util.comparators

import fobo66.exchangecourcesbelarus.entities.Currency

/**
 * Created by fobo66 on 31.03.2016.
 */
class RurSellComparator : Comparator<Currency> {
  override fun compare(lhs: Currency, rhs: Currency): Int {
    return rhs.rurSell.compareTo(lhs.rurSell, ignoreCase = true)
  }
}
